plugins/plugin-testing-java/plugin-testing-classpath/src/main/java/org/eclipse/che/plugin/java/testing/AbstractJavaTestRunner.java

```diff
@@ -30,17 +30,25 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.io.IOException;
+import java.net.Socket;
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
+import java.util.Random;
 
 /**
  * Abstract java test runner.
  * Can recognize test methods, find java project and compilation unit by path.
  */
 public abstract class AbstractJavaTestRunner implements TestRunner {
+    private static final   Logger LOG                  = LoggerFactory.getLogger(AbstractJavaTestRunner.class);
+    private static final   String TEST_OUTPUT_FOLDER   = "/test-output";
 
-    private static final Logger LOG = LoggerFactory.getLogger(AbstractJavaTestRunner.class);
+    protected static final String PROJECTS_ROOT_FOLDER = "/projects";
```

Comment:
Can we replace hrdcoded "/projects" with
`@Named("che.user.workspaces.storage") File projectsRoot ` ?